src/compas/geometry/_shapes/sphere.py

```diff
@@ -294,7 +294,7 @@ def transform(self, transformation):
         """
         self.point.transform(transformation)
         Sc, _Sh, _R, _Tl, _P = transformation.decomposed()
-        self.radius *= max(Sc.scale_factors)
+        self.radius *= max([Sc[0,0],Sc[1,1],Sc[2,2]])
```

Comment:
those underscored variables might as well be underscores if they are not used...

Reply:
Fixed